### PR TITLE
use official keycloak chart

### PR DESCRIPTION
**Pull request by ebiiii**
contributes to #208

### CODE REVIEW

File: charts/renku/values.yaml [hunk 1]
```diff
@@ -70,31 +70,48 @@ keycloak:
...
+      dbVendor: postgres
+
+      ## The following values only apply if "deployPostgres" is set to "false"
+
+      dbName: keycloak
+      dbHost: renku-postgresql
+      dbPort: 5432 # 5432 is PostgreSQL's default port. For MySQL it would be 3306
+      dbUser: keycloak
+
+      existingSecret: renku
```
Comment (leafty) [inline charts/renku/values.yaml]: TODO: Document the following helm template caveat (at least in `values.yaml`).

Convention in helm chart is to define name of resources using the following template:
```
{{- define "postgresql.fullname" -}}
{{- if .Values.fullnameOverride -}}
{{- .Values.fullnameOverride | trunc 20 | trimSuffix "-" -}}
{{- else -}}
{{- $name := default .Chart.Name .Values.nameOverride -}}
{{- if contains $name .Release.Name -}}
{{- .Release.Name | trunc 20 | trimSuffix "-" -}}
{{- else -}}
{{- printf "%s-%s" .Release.Name $name | trunc 20 | trimSuffix "-" -}}
{{- end -}}
{{- end -}}
{{- end -}}
```

This makes the resource name depend on the Helm release name.

The values of `dbHost` and `existingSecret` work when the release name is `renku`. As @ebiiii experienced, when deploying to a different release name, then at least `dbHost` and `existingSecret` need to be adjusted.

Comment (rokroskar) [reply charts/renku/values.yaml]: is there no way to define namespace-resolvable aliases that would make this work in general? Like network-specific aliases in `docker-compose`.

**Comment by rokroskar** [comment]
I tried out the PR on staging and it looks good, thanks @ebiiii -- modulo @leafty's request for the extra line of docs, this can be approved and merged.

File: charts/renku/values.yaml (hunk 1 above)
Comment (leafty) [reply charts/renku/values.yaml]: We could do by using `fullnameOverride`. If we want to have a preference for that, we can investigate it in a separate issue.

**Comment by rokroskar** [comment]
what is this PR waiting for?

**Comment by leafty** [comment]
@ebiiii need to manually set some values in https://github.com/SwissDataScienceCenter/renku/blob/master/scripts/travis-build.sh

**Pull request merged**